packages/angular/src/app/pages/planning-task-list/planning-task-list.component.html

```diff
@@ -127,4 +128,40 @@
     >
     </dx-load-panel>
   </ng-template>
+  <dx-popup
```

Comment:
For the user, we have a separate component. Why don't we have a separate component here?
Also, the popup configuration looks similar to user-popup. Looks like we can create a shared popup component to set different forms inside as children.

Reply:
Made separate component